subsys/bluetooth/services/ots/ots_oacp.c

```diff
@@ -23,7 +25,7 @@
 LOG_MODULE_DECLARE(bt_ots, CONFIG_BT_OTS_LOG_LEVEL);
 
 #define OACP_PROC_TYPE_SIZE	1
-#define OACP_RES_MAX_SIZE	3
+#define OACP_RES_MAX_SIZE	7
```

Comment:
Increased by 4 to support the 4-octet checksum?

Reply:
done